### PR TITLE
Add immature balance to UI

### PR DESCRIPTION
**Pull request by panleone**
## Abstract
---

The wallet class already ignores immature balance, but some users were confused about not being able to see their immature balance. This PR address this issue by adding immature balance both to dashboard and balance chart

---

## Testing
Any tips on how to improve the UI are welcome

### CODE REVIEW

File: assets/style/style.css [hunk 1]
```diff
@@ -2277,6 +2277,10 @@ a {
   font-size: 17px;
 }
 
+.dcWallet-balances .dcWallet-pivxImmatureBalance {
+    font-size: 17px;
+}
```
Comment (Duddino) [inline assets/style/style.css:2282]: You can add this directly to the .vue component with a `<style>` tag (You can then remove the `dcWallet` prefix, I'll eventually do this for all component specific css

File: scripts/mempool.js [hunk 2]
```diff
@@ -250,10 +257,10 @@ export class Mempool {
      * Get the total wallet balance
      * @param {UTXO_WALLET_STATE} filter the filter you want to apply
      */
-    getBalance(filter, includeLocked = false) {
+    getBalance(filter, includeLocked = false, includeImmature = false) {
```
Comment (Duddino) [inline scripts/mempool.js:260]: I don't like all these parameters, can you include these in the filter?

Comment (panleone) [reply scripts/mempool.js:260]: if I added them filter params would grow exponentially
SPENDABLE_LOCKED_IMMATURE
SPENDABLE_LOCKED_MATURE
SPENDABLE_UNLOCKED_IMMATURE
SPENDABLE_LOCKED_IMMATURE
.... and so on for cold staking filters...

I think that 3 params in getBalance are ok

Comment (panleone) [reply scripts/mempool.js]: ok I think I understood what you meant, review applied

**Review by Duddino — CHANGES_REQUESTED** [review]
Can you hide the balance when it's 0? 
Sorry for not saying earlier

**Comment by panleone** [comment]
> Can you hide the balance when it's 0? Sorry for not saying earlier

done

**Review by Duddino — APPROVED** [review]
tACK. Maybe the definition of immature balance could be further discussed, currently uncofirmed transactions are considered mature balance, but it's outside of the scope of this PR.

**Pull request merged**